subscriber/podaac_data_downloader.py

```diff
@@ -264,9 +266,23 @@ def run(args=None):
     logging.info("Found " + str(len(downloads)) + " total files to download")
     if download_limit:
         logging.info("Limiting downloads to " + str(args.limit) + " total files")
+
     if args.verbose:
         logging.info("Downloading files with extensions: " + str(extensions))
 
+    if args.dry_run:
+        logging.info("Dry-run option specified. Listing Downloads.")
+        if download_limit:
+            for x in range(0, min(download_limit, len(downloads))):
+                logging.info(downloads[x])
+        else:
+            for download in downloads:
+                logging.info(download)
```

Comment:
You can do this which works in all cases:

1. No download limit specified
2. Download limit specified and < num downloads
3. Download limit specified and > num downloads

```suggestion
        for download in downloads[:download_limit]:
            logging.info(download)
```